srtcore/buffer.cpp

```diff
@@ -804,12 +802,12 @@ int CRcvBuffer::readBuffer(char* data, int len)
             return -1;
         }
 
-        if (m_bTsbPdMode)
-        {
-            HLOGC(dlog.Debug, log << CONID() << "readBuffer: chk if time2play: NOW=" << now << " PKT TS=" << getPktTsbPdTime(m_pUnit[p]->m_Packet.getMsgTimeStamp()));
-            if ((getPktTsbPdTime(m_pUnit[p]->m_Packet.getMsgTimeStamp()) > now))
-                break; /* too early for this unit, return whatever was copied */
-        }
+      if (m_bTsbPdMode)
+      {
+          HLOGC(dlog.Debug, log << CONID() << "readBuffer: chk if time2play: PKT TS=" << FormatTime(getPktTsbPdTime(m_pUnit[p]->m_Packet.getMsgTimeStamp())));
```

Comment:
Note that a log always prints the time when its instruction is being executed.

Reply:
NOW should still be in the logs because it's not the time when the log is being printed, but locked in the beginning. Comparing it to the time of printing the log might be misleading, as slight difference may decide that a packet should not be played yet.